apps/nouns-camp/src/components/landing-screen.jsx

```diff
@@ -270,6 +270,11 @@ const BrowseScreen = () => {
 
   const query = searchParams.get("q") ?? "";
   const deferredQuery = React.useDeferredValue(query.trim());
+  const [inputValue, setInputValue] = React.useState(query);
+
+  React.useEffect(() => {
+    setInputValue(query);
+  }, [query]);
```

Comment:
this was required to have the search input sync with the `q` search param upon navigation